### ISSUE TITLE
Translation (Question)

### ISSUE THREAD

**Issue by Nizami20052022** [polymorphicshade/NewPipe#276]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/NewPipe/issues) or [closed](https://github.com/polymorphicshade/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/polymorphicshade/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

How do I do translations?

### Why do you want this feature?

I will add an Azerbaijani translation.

### Why ist the feature relevant to this fork?

Because the translations of SponsorBlock and additional features are in this fork.

**Comment by polymorphicshade** [comment]
Fork the repo, clone it to your local file system, add your translated strings into an appropriate XML file, then submit a pull-request.

More info:
- [Android Studio - Localize the UI with Translations Editor](https://developer.android.com/studio/write/translations-editor)
- [StackOverflow - Working with strings.xml and translations](https://stackoverflow.com/questions/47976576/working-with-strings-xml-and-translations)

Basically you just need a strings.xml specific to your target locale.
I believe the folder name would be "values-az" or something like that, not entirely sure though.
